app/components/client/playground-scripts.js

```diff
@@ -193,7 +193,7 @@ function getMemeInfo() { // TODO: Error handling
     language: document.getElementById("meme-language").value,
     license: document.getElementById("meme-license").value,
     name: document.getElementById("meme-title").value,
-    nsfw: document.getElementById("meme-nsfw-flag").checked,
+    tags: document.getElementById("meme-nsfw-flag").checked ? "mature" : "[]",
```

Comment:
Should this be `["mature"]`?

Reply:
that produces the same result in terms of the output. This only affects the suggested syntax on the command line:  ```"tags": mature```

The syntax does need to be cleaned up  now that we don't pass the metadata tag directly.